src/types.ts

```diff
@@ -34,3 +34,5 @@ export interface TileSubLabel {
     hasCancellation?: boolean
     hasSituation?: boolean
 }
+
+export type ThemeType = 'default' | 'dark' | 'light' | 'grey'
```

Comment:
Det kunne vore nyttig å bruke TypeScript sin `enum`-type for Theme og bruke det koden. Då kan du bruke `Theme.DARK` osv. for å vise til verdien, samtidig som du kan bruke `Theme` som type.

```
export enum Theme = {
    DEFAULT = 'default',
    DARK = 'dark',
    LIGHT = 'light',
    GREY = 'grey',
}
```